packages/hardhat/package.json

```diff
@@ -3,12 +3,14 @@
   "version": "0.0.1",
   "scripts": {
     "account": "hardhat run scripts/listAccount.ts",
+    "account:import": "hardhat run scripts/importAccount.ts",
+    "account:generate": "hardhat run scripts/generateAccount.ts",
```

Comment:
intuitively at the first try I entered `yarn account:generate <my_pk>` I think we need to add info how to use it to the docs. Since if people will use this `yarn account:generate <my_pk>` too, they will have their private key in terminal history. Also, we can mention that they need to clear clipboard after copying their PK to `account:generate`

Reply:
Agree, we need to update the docs!

Reply:
created an issue https://github.com/scaffold-eth/se-2-docs/issues/103

Reply:
Perfect, thanks!